resources/markdown/manual.md

```diff
@@ -1,36 +1,42 @@
-# **ProBro user manual** 
-This document provides guidelines how to use ProBro extension.
+# **ProBro user manual**
 
+This document provides guidelines how to use ProBro extension.
 
 ## **1. Opening ProBro extension:**
+
 ![Db Connection](../images/DbConnection.png)
 
- 1. ProBro extension image on sidebar.
- 2. Create new db connection. 
- 3. Refresh db connection list.
- 4. Delete existing connection.
- 5. Edit existing connection.
- 6. Group lists of databases. If no group is assigned to database, group is set to "Empty".
+1.  ProBro extension image on sidebar.
+2.  Create new db connection.
+3.  Refresh db connection list.
+4.  Delete existing connection.
+5.  Edit existing connection.
+6.  Launch data administration.
+7.  Launch data dictionary.
+8.  Launch procedure editor.
+9.  Refresh single connection.
+10. Group lists of databases. If no group is assigned to database, group is set to "Empty".
 
- Every db connection on start of extension or refreshing are testes if they are connected. Otherwise near the OpenEdge logo appears red sign.
+The icon indicates current status of the connection. Every db connection on the start of the extension or upon refreshing are tested to check if they are connected. Otherwise near the OpenEdge logo a red circle appears which indicates that the connection is unsuccessful (Disabled).
 
- ## **2. Creating new connection**
+## **2. Creating new connection**

```

Comment:
Add description on how connection import from `openedge-project.json` file works. This is primary workflow we suggest to our user, so it must be clearly defined.